scripts/build/builders/gn.py

```diff
@@ -95,6 +95,8 @@ def _build(self):
         self.PreBuildCommand()
 
         cmd = ['ninja', '-C', self.output_dir]
+        if self.ninja_jobs is not None:
+            cmd.append('-j' + str(self.ninja_jobs))
```

Comment:
how about parts that use ninja but are not gn?

scripts/build/builders/ameba.py
scripts/build/builders/tizen.py
scripts/build/builders/android.py
scripts/build/builders/nrf.py
scripts/build/builders/host.py
scripts/build/builders/telink.py

Reply:
Done.